t/Renard/Curie/Component/KeyBindings.t

```diff
@@ -0,0 +1,66 @@
+use Test::Most;
+use Test::More;
+
+use lib 't/lib/';
+use Modern::Perl;
+use Renard::Curie::App;
+use Renard::Curie::Model::Document::PDF;
+use CurieTestHelper;
+
+my $pdf_document = Renard::Curie::Model::Document::PDF->new(
+  filename => CurieTestHelper->test_data_directory->child(qw(PDF Adobe pdf_reference_1-7.pdf))
+);
+
+sub build_test{
```

Comment:
I took the `build_test` code and put it into `CurieTestHelper` as `run_app_with_document`. You can use that instead.